setup.py

```diff
@@ -1,8 +1,10 @@
 from setuptools import setup
 
+from sunpy_sphinx_theme import __version__
```

Comment:
this will break `pip install sunpy-sphinx-theme` in a clean env (test by `python setupy egg_info`)

Reply:
FINE MY GOD ITS NEVER GOOD ENOUGH FOR YOU